to_production.txt

```diff
@@ -1,24 +1,123 @@
 # New
 ofl/exile # https://github.com/google/fonts/pull/9281
+ofl/notoserifdivesakuru # https://github.com/google/fonts/pull/9047
 ofl/playpensansarabic # https://github.com/google/fonts/pull/9188
 ofl/playpensansdeva # https://github.com/google/fonts/pull/9304
 ofl/playpensanshebrew # https://github.com/google/fonts/pull/9187
 ofl/playpensansthai # https://github.com/google/fonts/pull/9198
+ofl/wdxllubrifonttc # https://github.com/google/fonts/pull/9380
```

Comment:
This family isn't ready yet. We need to work on the specimens that are shown on the front end.  Will need to remove from to_prod before we merge these lists.

cc @aaronbell @m4rc1e @emmamarichal